src/Site/views/languageforge/theme/default/sass/languageforge_default.scss

```diff
@@ -2,13 +2,10 @@
 @import '_variables.scss';
 
 // Pull in Bootstrap files - by default this loads ALL features of Bootstrap which we may not need
-@import '../../../../../../../../vendor_bower/bootstrap/scss/bootstrap.scss';
+@import '../.././../../../../vendor_bower/bootstrap/scss/bootstrap.scss';
```

Comment:
I thought we were removing this import.
and if there's a reason we're keeping this, there's a typo in the path (missing a '.')